shared/settings/advanced/index.js

```diff
@@ -1,35 +1,38 @@
 // @flow
 import * as React from 'react'
 import {globalStyles, globalMargins} from '../../styles'
-import {Box, Button, Text} from '../../common-adapters'
+import {StandardScreen, Box, Button, Text} from '../../common-adapters'
 
 type Props = {
   onDBNuke: () => void,
+  onBack: () => void,
 }
 
 function DBNuke(props: Props) {
   return (
-    <Box
-      style={{
-        ...globalStyles.flexBoxColumn,
-        alignItems: 'center',
-        flex: 1,
-        padding: globalMargins.medium,
-        paddingTop: globalMargins.xlarge,
-        paddingBottom: globalMargins.xlarge,
-      }}
-    >
-      <Text type="Body" style={{textAlign: 'center', padding: globalMargins.small}}>
-        Don't do anything here unless instructed to by a developer.
-      </Text>
-      <Button
-        style={{marginTop: globalMargins.small}}
-        type="Danger"
-        label="DB Nuke"
-        onClick={props.onDBNuke}
-      />
-      <Box style={{flex: 1}} />
-    </Box>
+    <StandardScreen onBack={props.onBack}>
```

Comment:
I think there might be an easier way, since we already have a `HeaderHoc` component. I think you wanna do something similar to `settings/about-container.js`, and have
```
const connectedDBNuke = compose(connect(mapStateToProps, mapDispatchToProps), HeaderHoc)(DBNuke)
export default connectedDBNuke
```

Does that sound right @MarcoPolo ?